oeps/oep-0001.rst

```diff
@@ -45,163 +61,211 @@ OEP Types
 * A **Process** proposal describes a change to how the Open edX community
   functions.
 
-* A **Best Practices** proposal describes a technology or implementation
+* A **Best Practice** proposal describes a technology or implementation
   choice that the Open edX community believes all applicable Open edX services
   and/or libraries should use or follow.
 
-* An **Architecture** proposal describes relationships between Open edX
-  services and libraries, including their APIs and data formats.
-
-* A **Product Enhancement** proposal describes a significant enhancement to the
-  functionality of the Open edX platform.
+* An **Architecture** proposal describes a structure or framework for Open
+  edX services or relationships between them.
 
-Workflow
-========
+OEP Roles
+=========
 
-.. contents::
-  :local:
-  :depth: 1
+Authors
+-------
 
-OEP Roles
----------
+Each OEP must have at least one Author: someone who writes the OEP using the
+style and format described here, shepherds the discussions in the appropriate
+forums, and attempts to build community consensus around the idea.  
 
-Each OEP must have a Author: someone who writes the OEP using the style and
-format described here, shepherds the discussions in the appropriate forums,
-and attempts to build community consensus around the idea.  
+Arbiter
+-------
 
-Each OEP also has an Arbiter (as described in `Initial Submission`_). The
-Arbiter will be chosen by the edX Chief Architect (currently Eddie Fagin).  
-The Author of an OEP will never be selected as the Arbiter of that OEP.
+Each OEP also has an Arbiter (as described in `Step 4. Request an Arbiter`_). 
+The Arbiter will be chosen by the `edX Architecture Team`_. An Author of an OEP
+will never be selected as the Arbiter of that OEP.
 
 The Arbiter will be the person making the final decision on whether the OEP 
 should be Accepted, and as such, the Arbiter should be knowledgeable about 
 the contents of the proposal, and willing to listen to arguments both for 
 and against it by the rest of the community.
 
-The Arbiter is also responsible for helping the Author move the proposal
+The Arbiter is also responsible for helping the Authors move the proposal
 through the OEP process, providing technical and process expertise as needed.
-The Arbiter also assists the Author to solicit feedback from the community on
+The Arbiter also assists the Authors to solicit feedback from the community on
 the OEP, and for helping to move the OEP towards a final decision (whether that
```

Comment:
Tiny grammatical nit, either subject/verb agreement here is weird or I'm parsing the compound predicate incorrectly.

- The Arbiter ... assists
- (The Arbiter) ... for helping to move

The simple fix is to replace `...and for helping to move...` with `...and helps to move...`. A potentially cleaner way to phrase the whole sentence would be `The Arbiter also assists the Authors in soliciting feedback from the community on the OEP and moving it towards a final decision (whether that decision is Accepted, Rejected, or Deferred).`

Reply:
fixed.